CheckValues/appsettings.test.json

```diff
@@ -0,0 +1,13 @@
+{
+    "Logging": {
+        "LogLevel": {
+            "Default": "Warning"
+        }
+    },
+    "AllowedHosts": "*",
+
+    "LiveConnectionString": "Data Source=.\\SQLEXPRESS;Initial Catalog=iot;Integrated Security=true",
+    "LocalLiveDBConnectionString": "Server=dev.retrotest.co.za;Database=iot;User Id=group1;Password=fNX^r+UKy3@CtYh5",
```

Comment:
This should not be there. 

The ideal in this case is that the live server has a copy of this file with its credentials in. These credentials should not be commited to the repo. If they are it should definitely be encrypted with keys that are not available and even then not to a public repo

Reply:
Please review this. I have made the changes. I am using Azure Variables in the pipeline to insert connectionstring into appsettings.json